docs/understanding_secret_sharing.rst

```diff
@@ -0,0 +1,115 @@
+Understanding Secret Sharing
+============================
+
+What is secret sharing?
+-----------------------
+
+Secret sharing refers to a family of cryptographic techniques
+for securely sharing a secret among a group of people.
+An abitrary secret, such as a passphrase or a cryptographic key,
+is split into a chosen number of secret shares
+such that a quorum of the shares may be used to recover the secret.
+The size of the group and the size of the quorum are configurable,
+and are chosen prior to generating the secret shares.
+
+For more detail and additional examples, see `Wikipedia`_.
+
+.. _Wikipedia: https://en.wikipedia.org/wiki/Secret_sharing
+
+.. _why-use-secret-sharing:
+
+Why would I want to use secret sharing?
+---------------------------------------
+
+Secret sharing is best suited for high-security use cases where a combination of
+one or more of the following properties is desired:
+
+-  **Security**: If you have a very important file or document whose
+   theft would be devastating, this tool might be of interest. By
+   splitting a secret into many shares, you make it exponentially harder
+   for an attacker to compromise the secret, because they have to
+   compromise several of the shares to recover the original.
+
+-  **Redundancy**: If you have a secret that must not be lost, it is
+   bad practice to keep it in only one place, or in only one person’s
+   head. By splitting a secret you can store the shares in many
+   different places—geographically or with different people--so that the
+   loss of one of the shares does not mean the loss of the secret. Doing
+   this with Sunder is better than simply copying the original secret
+   because the shareholders do not have individual access to the secret,
+   nor can the secret be stolen by compromising any individual backup
+   location.
+
+-  **Distributed trust**: Splitting a secret means that no person has
+   access to it when acting alone. Accessing the secret requires the
+   cooperation and consent of a configurable quorum of other people.
+
+When is secret sharing not appropriate?
+---------------------------------------
+
+- **Frequent access**: If you need to access the secret frequently,
+  daily or even weekly, this is probably not the right scheme for you.
+  Reconstituting the secret requires that a quorum of the shareholders
+  are present. If that is going to be a burden then you should
+  consider other options.
+
+.. _real-world-examples:
+
+What are some real world examples?
+----------------------------------
```

Comment:
Not listed in these examples are border crossing and media archival (e.g. for filmmakers). However, we have issues tracking those additions: #53 and #52, respectively, so fine to exclude for now.